islands/BotTokenInput.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx h  */
+import { h } from "preact";
+import { useEffect, useState } from "preact/hooks";
+import { tw } from "twind";
+
+export default function BotTokenInput() {
+  const [token, setToken] = useState("");
+
+  useEffect(() => {
+    setToken(localStorage.getItem("token") ?? "");
+  }, []);
+
+  useEffect(() => {
+    localStorage.setItem("token", token);
+  }, [token]);
+
+  return (
+    <input
+      type="password"
+      id="bot_token"
+      name="bot_token"
+      class={tw`bg-gray-50 border focus:outline-none border-gray-300 text-gray-900 rounded-lg focus:ring-grammy-500 focus:border-grammy-500 w-full p-2.5`}
```

Comment:
Browser logs

```
[UNKNOWN_THEME_VALUE] {"key":"ringColor.grammy-500"}
[UNKNOWN_THEME_VALUE] {"key":"borderColor.grammy-500"}
[UNKNOWN_DIRECTIVE] {"rule":"focus:border-grammy-500"}
```

on home page. Does it log for you?

Reply:
And I found it weird because yex.deno.dev (main) also had these classes, and it doesnt log anything like this.